php_phongo.c

```diff
@@ -963,7 +1001,12 @@ static bool php_phongo_apply_options_to_uri(mongoc_uri_t *uri, bson_t *options T
 			continue;
 		}
 
-		if (mongoc_uri_option_is_int32(key) && BSON_ITER_HOLDS_INT32(&iter)) {
+		if (mongoc_uri_option_is_int32(key)) {
+			if (!BSON_ITER_HOLDS_INT32(&iter)) {
+				PHONGO_URI_INVALID_TYPE(iter, "32-bit integer");
```

Comment:
Are you suggesting I create constants for all of the return values in `php_phongo_bson_type_to_string()`?

I don't necessarily agree with using constants here, as we also have `PHONGO_URI_INVALID_TYPE(iter, "array or object");` below. These strings are merely pieces of an exception message, just as we do in the [Query constructor](https://github.com/mongodb/mongo-php-driver/blob/223d1b91786ff8aefc7bc036f894d43835679cbf/src/MongoDB/Query.c#L135).